File: UI/LibraryAppUI/src/app/CustomClasses/Customer.ts
export class Customer{
    Email: string;
    Name: string;
    Sex: string;
    Address: string;
    DateOfBirth: Date;
    PhoneNumber: string;
    Password: string;
    Role: string;
    CustomerID: number;
//    issuedBooks: Array<IssuedBook> = new Array<IssuedBook>();
    constructor(_email: string, _name: string, _sex: string, _address: string,
        _dob: Date,_ph: string,_pswd: string, _role: string) {
        this.Email = _email;
        this.Name = _name;
        this.Sex = _sex;
        this.Address = _address;
        this.DateOfBirth = _dob;
        this.PhoneNumber = _ph;
        this.Password = _pswd;
        this.Role = _role;
    }
}

File: LibraryWebApplication/Models/Customer.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Models
{
    using System;
    using System.Collections.Generic;

    public partial class Customer
    {
        public int CustomerID { get; set; }
        public string UserID { get; set; }
        public string Name { get; set; }
        public string Sex { get; set; }
        public System.DateTime DateOfBirth { get; set; }
        public string Address { get; set; }
        public string PhoneNumber { get; set; }

        public virtual LibaryUser LibaryUser { get; set; }
    }
}
